packages/payment-processor/src/payment/erc777-stream.ts

```diff
@@ -47,11 +47,12 @@ export async function payErc777StreamRequest(
   // - use expectedStartDate to compute offset between start of invoicing and start of streaming
   // - start fee streaming
   const { paymentReference, paymentAddress, expectedFlowRate } = getRequestPaymentValues(request);
+  // To be able to filter SuperFluid subgraph events containing only this prefix
   const streamPayOp = sf.cfaV1.createFlow({
     flowRate: expectedFlowRate ?? '0',
     receiver: paymentAddress,
     superToken: superToken.address,
-    userData: `0x${paymentReference}`,
+    userData: `0xbeefac${paymentReference}`,
```

Comment:
Can you add a comment to explain how this prefix was generated? Is this just random or something that we normalized?

Reply:
I'm documenting it in the specs branch. For now I've just generated a prefix that's I've not seen yet in other streams events and I've tried to normalize it as some meaningful text using hexadecimal letters.

Reply:
Great! Thanks

Reply:
documented at https://github.com/RequestNetwork/requestNetwork/pull/688/commits/90e13c495929617a0e7091714515b09ae7c3d294